### PR TITLE
Added support for testing on postgres using docker

### PR DESCRIPTION
**Pull request by matthewhegarty**
Adds a `docker-compose.yml` file and supporting scripts to optionally test on Postgres.
This was added to test #102.

Incidentally, running `makemigrations` added a new migration for the example app.

**No changes to functional code**

### CODE REVIEW

File: docs/example-app.rst [hunk 1]
```diff
@@ -17,3 +17,32 @@ We encourage you to give it a try with a few commandline calls:
 A screenshot of the example app:
 
 .. image:: _static/screenshot.jpg
+
+Postgres
+########
+
+You can use Postgres as the source database.  To do this, you will need to have `Docker <https://docker.com/>`_ installed.
```
Comment (izimobil) [inline docs/example-app.rst:24]: You will need to have => You can use
Docker not being the only option to install postgres :)

File: example/albums/migrations/0002_auto_20210720_2132.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Generated by Django 3.2.5 on 2021-07-20 21:32
```
Comment (izimobil) [inline example/albums/migrations/0002_auto_20210720_2132.py:1]: Is this migration really needed ? I don't see any changes to the models of example app but maybe I'm missing something...

**Review by izimobil — CHANGES_REQUESTED** [review]

File: example/albums/migrations/0002_auto_20210720_2132.py (hunk 2 above)
Comment (matthewhegarty) [reply example/albums/migrations/0002_auto_20210720_2132.py:1]: It adds the 'ordering' (no idea why it didn't appear in the initial migration).  I can remove it if preferred - probably doesn't make any difference in the test app.

1st migration:

```
            options={
                'verbose_name': 'Album',
                'verbose_name_plural': 'Albums',
            },
```

new migration:

```
        migrations.AlterModelOptions(
            name='album',
            options={'ordering': ['name'], 'verbose_name': 'Album', 'verbose_name_plural': 'Albums'},
        ),
```

Comment (izimobil) [reply example/albums/migrations/0002_auto_20210720_2132.py:1]: OK thanks, we'll keep it

**Review by izimobil — APPROVED** [review]

**Comment by izimobil** [comment]
Merged, thanks !

**Pull request merged**